fairseq/sequence_generator.py

```diff
@@ -268,7 +272,7 @@ def get_hypo():
                 for i, model in enumerate(self.models):
                     if isinstance(model.decoder, FairseqIncrementalDecoder):
                         model.decoder.reorder_incremental_state(incremental_states[model], reorder_state)
-                    encoder_outs[i] = model.encoder.reorder_encoder_out(encoder_outs[i], reorder_state)
+                    encoder_outs[i] = model.decoder.reorder_encoder_out(encoder_outs[i], reorder_state)
```

Comment:
i thought they are all on encoder now, which maes sense.

in case of a language model encoder_outs should be empty anyway, maybe we can just check that? and this particular change would probably cause all the encoder-decoder models to crash (e.g. machine translation)

Reply:
oops yes I forgot we agreed to move them all into the encoder

Reply:
Yes, this should be `encoder` instead of `decoder`, I'll fix it.

@huihuifan , @alexeib Thank you so much for looking into this. I'll address all the comments and run test to make sure the changes doesn't break generation for translation.

Reply:
thanks!